syntax = "proto3";
package helloworld;

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply);
}

message HelloRequest {
  string name = 1;
  int32 age = 2;
  bool flag = 3;
}

message HelloReply {
  string message = 1;
}
